.storybook/preview.js

```diff
@@ -0,0 +1,23 @@
+import '../src/openforms/scss/admin/admin_overrides.scss';
+import '../src/openforms/scss/screen.scss';
+import '../static/admin/css/admin-index.css';
```

Comment:
Can you add a comment specifying this needs `collectstatic`? Also add a storybook section to the developer docs documenting this requirement (and general usage of course)

Reply:
Added this and modified the GH action as well